content/docs/guides/neon-private-access.md

```diff
@@ -0,0 +1,97 @@
+
+![Neon PrivateLink configuration diagram](/docs/guides/privatelink.png)
+
+## Prerequisites
+
+- Ensure that your **client application is deployed on AWS** in the same region as the Neon database you plan to connect to. For the Private Preview of the **Neon Private Access** feature, only the AWS **us-east-1** and **eu-central-1** regions are supported. Your private access client application and Neon database must reside in one of these regions.
+- You will need to add a [VPC endpoint](https://docs.aws.amazon.com/vpc/latest/privatelink/concepts.html#concepts-vpc-endpoints) to the AWS Virtual Private Cloud ([VPC](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html)) where your client application is deployed. The steps are outlined below.
+
+## Configuration steps
+
+To configure Neon Private Access, perform the following steps:
+
+1. **Create an AWS VPC endpoint**
+
+   1. Go to the **AWS VPC Dashboard** and select **Create endpoint**. Make sure you create the endpoint in the same VPC as your client application.
+
+      ![VPC Dashboard](/docs/guides/pl_vpc_dashboard.png)
+
+   1. Optionally, enter a **Name tag** for the endpoint (e.g., `My Neon Private Access test`).
+   1. For **Service category**, select **Other endpoint services**.
+   1. Specify the **Service name**. It must be one of the following names, depending on your region:
+
+      - **us-east-1**: `com.amazonaws.vpce.us-east-1.vpce-svc-0ccf08d7888526333`
+      - **eu-central-1**: `com.amazonaws.vpce.eu-central-1.vpce-svc-0fa74d33d011f0803`
+
+      ![Select the endpoint service](/docs/guides/pl_select_endpoint_service.png)
+
+   1. Click **Verify service**. If successful, you should see a `Service name verified` message.
+   1. Select the VPC where your application is deployed.
+   1. Add the availability zones and associated subnets you want to support.
+   1. Click **Create endpoint** to complete the setup of the endpoint service.
+
+2. **Provide the endpoint ID to Neon**
+
+   Note the ID of the created endpoint and provide it to Neon. Neon will authorize this endpoint to access the Neon Private Access service and notify when the authorization is complete.
+
+3. **Enable Private DNS**
+
+   After Neon authorizes your endpoint (please wait for confirmation from Neon), you need to enable private DNS lookup for the endpoint.
+
+   1. In AWS, select the VCP endpoint you created.
```

Comment:
This must be VPC instead of VCP - my bad.

Reply:
fixed.